persistence/sql/migrations/sql/20230216142104000000_session_devices_index_drop.up.sql

```diff
@@ -0,0 +1 @@
+DROP INDEX IF EXISTS session_devices@unique_session_device CASCADE;
```

Comment:
not sure if this syntax works across all databases

Reply:
fixed